source/03-components/_index.scss

```diff
@@ -1,4 +1,6 @@
 @use '00-cms-styles/toolbar-menu/toolbar-menu';
+@use 'accordion/accordion';
+@use 'accordion-item/accordion-item';
```

Comment:
If we end up having accordion styles be in its own stylesheet rather than in the main one, these can be removed.